framework/core/js/src/forum/components/PostStream.js

```diff
@@ -78,11 +78,22 @@ export default class PostStream extends Component {
         content = <LoadingPost />;
       }
 
-      return (
+      const postStreamElement = (
         <div className="PostStream-item" {...attrs}>
           {content}
         </div>
       );
+
+      if (post && post.number() === 1 && this.afterFirstPostItems().toArray().length > 0) {
```

Comment:
I don't trust the first post number to always be 1, a more certain check would be using the discussion's first_post_id value, can we serialize that then use it hear?

Reply:
Actually in 2.0 the relationship is always "linked" so no need for extra serialization, we can do:

```suggestion
      if (post && post.id() === this.discussion.data.relationships.firstPost?.data.id && this.afterFirstPostItems().toArray().length > 0) {
```